manifests/e/Enigma/VirtualBox/11.00/Enigma.VirtualBox.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Enigma.VirtualBox
+PackageVersion: '11.00'
+InstallerLocale: en-US
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: 湅杩慭嘠物畴污䈠硯_is1
+ReleaseDate: 2024-08-26
+AppsAndFeaturesEntries:
+- Publisher: 桔⁥湅杩慭倠潲整瑣牯䐠癥汥灯牥⁳敔浡
+  ProductCode: 湅杩慭嘠物畴污䈠硯_is1

```

Comment:
Are these supposed to be chinese characters?

Reply:
This doesn't seem to match the Registry:
![image](https://github.com/user-attachments/assets/fd10d1b8-e41d-4f3c-8057-70f09a074e92)